.gitignore

```diff
@@ -179,3 +179,4 @@ dmypy.json
 # Mac OS files
 .DS_Store
 .vim
+models
```

Comment:
What's this ignore for?

Reply:
This was because sometimes I run the app locally (not in docker) and link in the models that can be downloaded (see https://github.com/climatepolicyradar/navigator/blob/e28ef9b45a4366231c4510852111a5e6ac3dadac/backend/Dockerfile#L22)

Reply:
ah ok, so this only ignores `navigator/models`?

Reply:
I've made it specific now ;)